Installation/CHANGES.md

```diff
@@ -29,6 +29,10 @@ Release date: June 2022
     [`remove_isolated_vertices()`](https://doc.cgal.org/5.5/Mesh_3/classCGAL_1_1Mesh__complex__3__in__triangulation__3.html#ace57c4e777da457c6e33b4f6e89949ce)
     as a post-processing step for the tetrahedral mesh generation.
 
+### [2D Polygons](https://doc.cgal.org/5.5/Manual/packages.html#PkgPolygon3)
```

Comment:
Should this be:
`https://doc.cgal.org/5.5/Manual/packages.html#PkgPolygon2`

Reply:
By the way, I have reactivated the redirection, so that a link like https://doc.cgal.org/5.5/Manual/packages.html#PkgPolygon2 points redirects to the equivalent in the documentation of `master`.